enterprise/frontend/src/embedding-sdk/components/public/StaticQuestion.tsx

```diff
@@ -86,18 +82,8 @@ export const StaticQuestion = ({
   };
 
   useEffect(() => {
-    if (!isInitialized || !isLoggedIn) {
-      setState({
-        loading: false,
-        card: null,
-        result: null,
-        cardError: null,
-        resultError: null,
-      });
-    } else {
-      loadCardData({ questionId });
-    }
-  }, [isInitialized, isLoggedIn, questionId]);
+    loadCardData({ questionId });
```

Comment:
Since we're separating the concern of logging in from the component, we can just load the question once the component is actually rendered

Reply:
We wouldn't allow users to completely replace `PublicComponentWrapper`, but the loading and error components that are in the wrapper. So the clients will be able to customize the appearance of the loading states, error messages, etc. but the general logic of the wrapper component will always be there.

So with this component for example, `StaticQuestion` will _only_ load once the login status is `success`, at which point the API will have been connected and things _should_ be good to go. Otherwise it's loading/error states